pkg/apis/externaldns/types.go

```diff
@@ -194,6 +198,8 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("publish-internal-services", "Allow external-dns to publish DNS records for ClusterIP services (optional)").BoolVar(&cfg.PublishInternal)
 	app.Flag("publish-host-ip", "Allow external-dns to publish host-ip for headless services (optional)").BoolVar(&cfg.PublishHostIP)
 	app.Flag("connector-source-server", "The server to connect for connector source, valid only when using connector source").Default(defaultConfig.ConnectorSourceServer).StringVar(&cfg.ConnectorSourceServer)
+	app.Flag("generic-source-crd-apiversion", "Api version of the crd for generic source, format (group/version) valid only when using generic source").Default(defaultConfig.GenericSourceCRDApiVer).StringVar(&cfg.GenericSourceCRDApiVer)
+	app.Flag("generic-source-crd-kind", "Kind of the crd for generic source, valid only when using generic source").Default(defaultConfig.GenericSourceCRDKind).StringVar(&cfg.GenericSourceCRDKind)
```

Comment:
Suggestion, make the description here: `Kind of the CRD for the generic source in API group and version specified by generic-source-crd-apiversion`

Reply:
will update.